docs/core/howto/tutorial/intro.rst

```diff
@@ -395,29 +395,18 @@ server does.
 Read Status from the Web
 ------------------------
 
-
-
-The web. That invention which has infiltrated homes around the
-world finally gets through to our invention. In this case we use the
-built-in Twisted web client
-via ``twisted.web.client.getPage`` , a non-blocking version of
-Python's :func:`urllib2.urlopen(URL).read <urllib2.urlopen>` .
-Like ``getProcessOutput`` it returns a Deferred which will be
-called back with a string, and can thus be used as a drop-in
-replacement.
-
-
-
+The web. That invention which has infiltrated homes around the world finally
+gets through to our invention. In this case we use the built-in Twisted web
+client via ``twisted.web.client.getPage`` , a non-blocking version of Python's
+:func:`urllib.urlopen(URL).read <urllib.request.urlopen>` .  Like
+``getProcessOutput`` it returns a Deferred which will be called back with a
+string, and can thus be used as a drop-in replacement.
```

Comment:
This is ok. Just a minor comment.
Maybe when we reformat the lines, we can also update to
[semantic line breaks](https://sembr.org/).

Reply:
I am going to skip that in this PR because I cannot figure out how to get my editor's paragaph wrapping (Emacs's fill-paragraph) to properly respect these line breaks, or have any kind of linting ensure them.  I would like to figure out a systematic way to do this though.